### DISCUSSION TITLE
Shared scrollback across multiple windows

### DISCUSSION THREAD

**Discussion by thenbe** [kovidgoyal/kitty#6548] [Q&A]
I'm looking for a way to set up my terminal (kitty) to display the output of a single command across multiple vertically split windows. Here's a visual representation of what I'm trying to achieve:

```
+-----------------+-----------------+-----------------+
|                 |                 |                 |
|    Window 1     |    Window 2     |    Window 3     |
|                 |                 |                 |
|  $ command      |  line 4: output |  line 7: output |
|  line 1: output |  line 5: output |  line 8: output |
|  line 2: output |  line 6: output |  line 9: output |
|  line 3: output |                 |                 |
|                 |                 |                 |
+-----------------+-----------------+-----------------+
```

Does anything similar exist? If not, does anyone have any sort of advice on where to start to implement such functionality?

Why? In many situations, this layout would be very useful (e.g. `tail`ing logs on a wide monitor).

Thank you.

**Comment by kovidgoyal** [comment]
You want a multi column view of a text file? Then the best way to
implement that is to write a pager like less that displays the text file
in as many columns as you like.

And there will likely be lots of people that have built these already
for you, for example a quick goole leads me to: https://github.com/ivanov/kanten
